Packs/Core/Playbooks/playbook-Unsigned_and_unpopular_process_performed_an_injection.yml

```diff
@@ -0,0 +1,823 @@
+      version: -1
+      name: Done
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 770,
+          "y": 2780
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "10":
+    id: "10"
+    taskid: 358ad811-3ae6-4e1d-826e-ba15c09f050c
+    type: title
+    task:
+      id: 358ad811-3ae6-4e1d-826e-ba15c09f050c
+      version: -1
+      name: Remediation
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "18"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 90,
+          "y": 1490
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "12":
+    id: "12"
+    taskid: a4e84519-ae9c-4cde-86db-4210bd57a617
+    type: condition
+    task:
+      id: a4e84519-ae9c-4cde-86db-4210bd57a617
+      version: -1
+      name: Approve the endpoint isolation
+      description: Requires the analyst approval for endpoint isolation.
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#default#':
+      - "8"
+      Isolate:
+      - "13"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": -300,
+          "y": 2240
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "13":
+    id: "13"
+    taskid: 324312f8-a792-4ff6-8046-848f554bdf15
+    type: regular
+    task:
+      id: 324312f8-a792-4ff6-8046-848f554bdf15
+      version: -1
+      name: Isolate endpoint
+      description: Isolates the specified endpoint.
+      script: '|||core-isolate-endpoint'
+      type: regular
+      iscommand: true
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "8"
+    scriptarguments:
+      endpoint_id:
+        simple: ${alert.agentid}
+      incident_id:
+        simple: ${parentIncidentFields.incident_id}
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": -300,
+          "y": 2430
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "14":
+    id: "14"
+    taskid: 5e10c74a-e684-4d52-8131-45f0d93e265e
+    type: condition
+    task:
+      id: 5e10c74a-e684-4d52-8131-45f0d93e265e
+      version: -1
+      name: Should remediate the threat?
```

Comment:
Amazing description! 
because we only going to terminate the CGP (the causality is not sign), lets rename this task name to should terminate the CGO? causality (CGO)

Reply:
Done